src/controllers/update/overseas.entity.change.controller.ts

```diff
@@ -49,3 +52,40 @@ export const post = async (req: Request, resp: Response, next: NextFunction) =>
     next(errors);
   }
 };
+
+export const resetChangeData = async (req: Request, appData: ApplicationData) => {
+  if (appData && appData.update){
```

Comment:
I'm not sure appData.update is defined when we get to here. Safer to do the appData and appData.update in separate if statements?